Build failure: ueviewer

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

```
environment = {
  systemPackages = with pkgs; [
    ueviewer
  ];
};
```

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/295083954

### Relevant log output

```console
Running phase: unpackPhase
unpacking source archive /nix/store/d267jw448z3m95n7nibmqayzz1mp6z8s-source
source root is source
Running phase: patchPhase
applying patch /nix/store/nmbg7a89myc8j7py48df5qa9cc2q8sxx-0001-ueviewer-Dont-use-c++2a-standard.patch
patching file common.project
patching script interpreter paths in build.sh Unreal/Shaders/make.pl Tools/genmake
build.sh: interpreter directive changed from "#!/bin/bash" to "/nix/store/9nw8b61s8lfdn8fkabxhbz0s775gjhbr-bash-5.2p37/bin/bash"
Unreal/Shaders/make.pl: interpreter directive changed from "#!/usr/bin/perl -w" to "/nix/store/nkn5ylz22hb1069w8q7n76dacm6jkw4k-perl-5.40.0/bin/perl -w"
" to "/nix/store/nkn5ylz22hb1069w8q7n76dacm6jkw4k-perl-5.40.0/bin/perl -w"
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
>> Debug: Detected platform: linux64
Unreal/Shaders/../Shaders.h does not exists, creating ...
>> Debug: Using makefile: ./obj/umodel-linux64.mak
Generating makefile from UmodelTool/umodel.project ...
./libs/oodle/lib/Linux/liboo2corelinux64.a doesn't exist
+ mkdir -p ./obj/libs-unix64
+ mkdir -p ./obj/umodel-unix64
+ echo Export3D.cpp
Export3D.cpp
+ echo ExportGLTF.cpp
ExportGLTF.cpp
+ echo Exporters.cpp
Exporters.cpp
+ echo ExportMaterial.cpp
ExportMaterial.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/Export3D.o Exporters/Export3D.cpp
+ echo ExportMd5.cpp
ExportMd5.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/ExportGLTF.o Exporters/ExportGLTF.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/ExportMaterial.o Exporters/ExportMaterial.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/ExportMd5.o Exporters/ExportMd5.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/Exporters.o Exporters/Exporters.cpp
+ echo ExportPsk.cpp
ExportPsk.cpp
+ echo ExportSound.cpp
ExportSound.cpp
+ echo ExportTexture.cpp
ExportTexture.cpp
+ echo ExportThirdParty.cpp
ExportThirdParty.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/ExportSound.o Exporters/ExportSound.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/ExportPsk.o Exporters/ExportPsk.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/ExportTexture.o Exporters/ExportTexture.cpp
+ echo GameDatabase.cpp
GameDatabase.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/ExportThirdParty.o Exporters/ExportThirdParty.cpp
+ echo TypeInfo.cpp
TypeInfo.cpp
+ echo UnCore.cpp
UnCore.cpp
+ echo UnCoreCompression.cpp
UnCoreCompression.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/GameDatabase.o Unreal/GameDatabase.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/TypeInfo.o Unreal/TypeInfo.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/UnCore.o Unreal/UnCore.cpp
+ echo UnCoreDecrypt.cpp
UnCoreDecrypt.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/UnCoreCompression.o Unreal/UnCoreCompression.cpp
+ echo UnCoreSerialize.cpp
UnCoreSerialize.cpp
+ echo UnObject.cpp
UnObject.cpp
+ echo UnObject4.cpp
UnObject4.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/UnCoreDecrypt.o Unreal/UnCoreDecrypt.cpp
+ echo UnRenderer.cpp
UnRenderer.cpp
+ echo FileSystemUtils.cpp
FileSystemUtils.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/UnCoreSerialize.o Unreal/UnCoreSerialize.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/UnObject.o Unreal/UnObject.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/UnObject4.o Unreal/UnObject4.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/FileSystemUtils.o Unreal/FileSystem/FileSystemUtils.cpp
+ echo GameFileSystem.cpp
GameFileSystem.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/UnRenderer.o Unreal/UnRenderer.cpp
+ echo GameFileSystemGears4.cpp
GameFileSystemGears4.cpp
+ echo IOStoreFileSystem.cpp
IOStoreFileSystem.cpp
+ echo UnArchivePak.cpp
UnArchivePak.cpp
+ echo UnHavok.cpp
UnHavok.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/GameFileSystem.o Unreal/FileSystem/GameFileSystem.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/GameFileSystemGears4.o Unreal/FileSystem/GameFileSystemGears4.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/IOStoreFileSystem.o Unreal/FileSystem/IOStoreFileSystem.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/UnHavok.o Unreal/GameSpecific/UnHavok.cpp
+ g++ -m64 -c -msse2 -fno-strict-aliasing -fno-stack-protector -Wunused-value -Os -std=c++14 -Wno-invalid-offsetof -I . -I ./Core -I ./Unreal -I ./libs/include -I ./libs -I ./libs/nvtt -I /usr/include/libpng -I ./libs/PowerVR -I ./libs/detex -I ./libs/rijndael -I UmodelTool -I UI -o ./obj/umodel-unix64/UnArchivePak.o Unreal/FileSystem/UnArchivePak.cpp
In file included from ./Unreal/UnCore.h:6,
                 from Exporters/ExportMaterial.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from ./Unreal/UnCore.h:6,
                 from Exporters/Export3D.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from Unreal/UnCore.h:6,
                 from Unreal/GameDatabase.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from ./Unreal/UnCore.h:6,
                 from Unreal/FileSystem/FileSystemUtils.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from Unreal/UnCore.h:6,
                 from Unreal/TypeInfo.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from Unreal/UnCore.h:6,
                 from Unreal/UnCore.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from ./Unreal/UnCore.h:6,
                 from Exporters/ExportGLTF.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from Unreal/UnCore.h:6,
                 from Unreal/UnCoreSerialize.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from ./Unreal/UnCore.h:6,
                 from Unreal/FileSystem/IOStoreFileSystem.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
make: *** [obj/umodel-linux64.mak:856: obj/umodel-unix64/ExportMaterial.o] Error 1
make: *** Waiting for unfinished jobs....
In file included from Unreal/UnCore.h:6,
                 from Unreal/UnRenderer.cpp:3:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from ./Unreal/UnCore.h:6,
                 from Unreal/FileSystem/GameFileSystem.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from ./Unreal/UnCore.h:6,
                 from Exporters/ExportSound.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from ./Unreal/UnCore.h:6,
                 from Unreal/FileSystem/UnArchivePak.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
make: *** [obj/umodel-linux64.mak:876: obj/umodel-unix64/Export3D.o] Error 1
In file included from ./Unreal/UnCore.h:6,
                 from Exporters/ExportMd5.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from Unreal/UnCore.h:6,
                 from Unreal/UnObject.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from ./Unreal/UnCore.h:6,
                 from Exporters/ExportPsk.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
make: *** [obj/umodel-linux64.mak:1137: obj/umodel-unix64/GameDatabase.o] Error 1
make: *** [obj/umodel-linux64.mak:1441: obj/umodel-unix64/TypeInfo.o] Error 1
make: *** [obj/umodel-linux64.mak:1582: obj/umodel-unix64/FileSystemUtils.o] Error 1
In file included from Unreal/UnCore.h:6,
                 from Unreal/UnCoreCompression.cpp:7:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
make: *** [obj/umodel-linux64.mak:705: obj/umodel-unix64/UnCore.o] Error 1
In file included from ./Unreal/UnCore.h:6,
                 from Exporters/Exporters.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
make: *** [obj/umodel-linux64.mak:776: obj/umodel-unix64/ExportGLTF.o] Error 1
make: *** [obj/umodel-linux64.mak:689: obj/umodel-unix64/UnCoreSerialize.o] Error 1
make: *** [obj/umodel-linux64.mak:1026: obj/umodel-unix64/IOStoreFileSystem.o] Error 1
make: *** [obj/umodel-linux64.mak:551: obj/umodel-unix64/UnRenderer.o] Error 1
make: *** [obj/umodel-linux64.mak:817: obj/umodel-unix64/ExportSound.o] Error 1
make: *** [obj/umodel-linux64.mak:669: obj/umodel-unix64/GameFileSystem.o] Error 1
make: *** [obj/umodel-linux64.mak:1044: obj/umodel-unix64/UnArchivePak.o] Error 1
make: *** [obj/umodel-linux64.mak:797: obj/umodel-unix64/ExportMd5.o] Error 1
make: *** [obj/umodel-linux64.mak:1099: obj/umodel-unix64/UnObject.o] Error 1
make: *** [obj/umodel-linux64.mak:734: obj/umodel-unix64/ExportPsk.o] Error 1
make: *** [obj/umodel-linux64.mak:1628: obj/umodel-unix64/UnCoreCompression.o] Error 1
make: *** [obj/umodel-linux64.mak:619: obj/umodel-unix64/Exporters.o] Error 1
In file included from Unreal/UnCore.h:6,
                 from Unreal/UnObject4.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from ./Unreal/UnCore.h:6,
                 from Exporters/ExportThirdParty.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
make: *** [obj/umodel-linux64.mak:1445: obj/umodel-unix64/UnObject4.o] Error 1
In file included from ./Unreal/UnCore.h:6,
                 from Exporters/ExportTexture.cpp:4:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
In file included from ./Unreal/UnCore.h:6,
                 from Unreal/FileSystem/GameFileSystemGears4.cpp:2:
./Core/CoreGL.h:9:10: fatal error: GL/gl.h: No such file or directory
    9 | #include <GL/gl.h>
      |          ^~~~~~~~~
compilation terminated.
make: *** [obj/umodel-linux64.mak:836: obj/umodel-unix64/ExportThirdParty.o] Error 1
make: *** [obj/umodel-linux64.mak:599: obj/umodel-unix64/ExportTexture.o] Error 1
make: *** [obj/umodel-linux64.mak:1079: obj/umodel-unix64/GameFileSystemGears4.o] Error 1
```

### Additional context

Not the most essential package, ofc, opening the issue mostly due to possible implications of dependencies' problems.

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.14.2, NixOS, 25.05 (Warbler), 25.05.20250412.2631b0b`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.1`
 - nixpkgs: `/nix/store/5ds20jm3x2s4z7wn3581r6lc9ybmh45b-source`

### Notify maintainers

@OPNA2608

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ue/ueviewer/package.nix
+++ b/pkgs/by-name/ue/ueviewer/package.nix
@@ -4,7 +4,9 @@
   fetchFromGitHub,
   fetchpatch,
   unstableGitUpdater,
+  libGL,
   libpng,
+  libX11,
   perl,
   SDL2,
   zlib,
@@ -62,6 +64,8 @@ gccStdenv.mkDerivation (finalAttrs: {
       zlib
     ]
     ++ lib.optionals (!gccStdenv.hostPlatform.isDarwin) [
+      libGL
+      libX11
       SDL2
     ];
 

